code/device/viewer.h

```diff
@@ -70,6 +74,10 @@ class viewer: public QGraphicsView
 
     std::map<QString, QGraphicsSvgItem*> unstableVisibleItems;
     std::map<QString, QGraphicsSvgItem*> selectableItems;
+
+    QGraphicsSvgItem* recoloredItem;    // Later I will make it to vector. IDD
+    QColor highlightColor = Qt::cyan;   // Color of highlighting. IDD
```

Comment:
static const?

Reply:
I can't set Qcolor as static or const because Qt swears me. 
Making recoloredItem to static will not have any effect.